src/main.c

```diff
@@ -392,6 +397,15 @@ int main(int argc, char **argv)
 				cli_cfg.user_key = strdup(optarg);
 				break;
 			}
+			if (strcmp(long_options[option_index].name,
+			           "user-key-passphrase") == 0) {
+				cli_cfg.user_key_passphrase = strdup(optarg);
```

Comment:
line 402 shouldn't be there. Without that it works. line 403 is the correct one

Reply:
I think the intention of `strdup()` on L. 402 is to allocate the buffer that is passed to `strncpy()` on L. 403. 
Unfortunately, the code doesn't compile 
```
src/main.c: In function ‘main’:
src/main.c:402:33: error: assignment to expression with array type
     cli_cfg.user_key_passphrase = strdup(optarg);
                                 ^
Makefile:620: recipe for target 'src/openfortivpn-main.o' failed
make: *** [src/openfortivpn-main.o] Error 1
```
vpn_config uses static arrays, so allocation of a fresh buffer isn't needed. We can just copy the string into cli_cfg.user_key_passphrase

Reply:
the current version works now, including the command line option